### PR TITLE
Support CPP layout without symlinks

### PR DESCRIPTION
**Pull request by finomen**

### CODE REVIEW

File: RULES/core/main.go [hunk 1]
```diff
@@ -26,6 +26,7 @@ type generatorInput struct {
 	CompletionsOnly bool
 	RunArgs         []string
 	TestArgs        []string
+	Layout          string
```
Comment (jfrohnhofen) [inline RULES/core/main.go:29]: What happens when we have a workspace with mixed layout?

File: RULES/core/path.go [hunk 2]
```diff
@@ -17,13 +17,21 @@ type Path interface {
 	WithSuffix(suffix string) OutPath
 }
 
+type PkgPath interface {
```
Comment (jfrohnhofen) [inline RULES/core/path.go:20]: who will implement that interface?

Comment (finomen) [reply RULES/core/path.go:20]: It's in generated file from dbt

File: RULES/core/main.go (hunk 1 above)
Comment (finomen) [reply RULES/core/main.go]: Removed, this is not used

**Pull request merged**